### DISCUSSION TITLE
Change mouse cursor size

### DISCUSSION THREAD

**Discussion by qorenta** [i3/i3#6074] [Q&A]
How can I increase the cursor size ? I use i3 with ubuntu 22.04
I already tried adding 
"exec_always --no-startup-id xcursor-size 48"
to my .config/i3/config, I also made a Xresources with Xcursor.size but it also didn't work.

**Comment by orestisfl** [comment]
>  "exec_always --no-startup-id xcursor-size 48"

I don't think there is an xcursor-size command so that wouldn't work. Test it in your terminal first before you add it in your i3 config.

Give https://wiki.archlinux.org/title/Cursor_themes a read, not all cursor themes support changing size. I use the Adwaita theme and added
```
Xcursor.size: 48
```
to my ~/.Xresources and it works.

**Comment by bancobusto** [comment]
I'm not certain if this will help you in your use case, but for me at the bottom of my .xinitrc I have:

xrdb -merge ~/.Xresources
exec i3

And in my .Xresources file, I have put:

Xcursor.theme: Bibata-Modern-Ice
Xcursor.size: 32

Obviously adjusted for the size of cursor, if your cursor supports the scaling etc.  I hope that this helps, best wishes. 
Mike
